docs/operations/hotplug_interfaces.md

```diff
@@ -132,10 +132,9 @@ limitations: each interface will consume a PCI slot in the VM, and there are a
 total maximum of 32. Furthermore, other devices will also use these PCI slots
 (e.g. disks, guest-agent, etc).
 
-For the `q35` machine type, the issue is even worse: the users will be able to
-hotplug a **single** network interface into the guest. You can find more
-information in the
-[libvirt documentation](https://libvirt.org/pci-hotplug.html#q35-machine-type).
+Kubevirt reserves resources for 4 interface to allow later hotplug operations.
+The actual maximum amount of available resources depends on the machine
+type (e.g. q35 adds another PCI slot).
```

Comment:
Please consider re-adding the link to libvirt's documentation.

Reply:
I do not see how this low level detail from libvirt is now relevant.
We are looking to abstract the detail with this solution and I prefer not to delve the reader into such details.

Reply:
> I do not see how this low level detail from libvirt is now relevant.

There is a sentence talking about differences between machine types.

I see this sentence and the link to the libvirt documentation as notes for further reading to someone that wants to dive deeper into the details.

Reply:
I think it is wrong to add this to the user-guide, it exposes readers to information which kubevirt is trying to abstract. But it is not worth the effort of arguing about it further.

Done.